src/server/pegasus_server_impl.h

```diff
@@ -382,6 +398,8 @@ class pegasus_server_impl : public pegasus_read_service
     uint64_t _abnormal_get_size_threshold;
     uint64_t _abnormal_multi_get_size_threshold;
     uint64_t _abnormal_multi_get_iterate_count_threshold;
+    uint64_t _abnormal_batch_get_size_threshold;
```

Comment:
define `gflag` and consider  if support `auto-update`

Reply:
(1)where to change the `gflag`? I’m not sure what you said; 
(2)I have considered  supporting `auto-update`, but gave up finally and consequently let these two added `threshold` just behave in consistent with `_abnormal_multi_get_size_threshold`  or `_abnormal_multi_get_iterate_count_threshold`;  
we will change these two `threshold` through config-file only;

Reply:
using gflags define: https://github.com/XiaoMi/rdsn/blob/master/src/nfs/nfs_client_impl.cpp#L40
using the variable defined: https://github.com/XiaoMi/rdsn/blob/master/src/nfs/nfs_client_impl.cpp#L54
support auto-update: https://github.com/XiaoMi/rdsn/blob/master/src/nfs/nfs_client_impl.cpp#L49

`auto-update` is optional, you can select not enable it

Reply:
ok